src/pages/builder/Builder.jsx

```diff
@@ -1,3 +1,170 @@
+import GjsEditor from '@grapesjs/react';
+import gjsForms from 'grapesjs-plugin-forms';
+import './customStyles.module.css'
+
+const gjsOptions = {
+    height: '100vh',
+    storageManager: {
+      type: 'remote',
+      stepsBeforeSave: 1,
+      options: {
+        remote: {
+          urlStore: 'http://127.0.0.1:3000/data', // Endpoint URL where to store data project
+          //  urlLoad: 'https://your-server/endpoint/load', // Endpoint URL where to load data project
+          onStore: (data) => {
+            console.log('onStore', data);
+            return { data}
+          },
+        },
+      },
+      autosave: true,
+      autoload: true,
+      contentTypeJson: true,
+      storeComponents: true,
+      storeStyles: true,
+      storeHtml: true,
+      storeCss: true,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      json_encode: {
+        'gjs-html': [],
+        'gjs-css': [],
+      },
+    },
+    undoManager: { trackSelection: false },
+    selectorManager: { componentFirst: true },
+  
+    projectData: {
+      assets: [
+        'https://via.placeholder.com/350x250/78c5d6/fff',
+        'https://via.placeholder.com/350x250/459ba8/fff',
+        'https://via.placeholder.com/350x250/79c267/fff',
+        'https://via.placeholder.com/350x250/c5d647/fff',
+        'https://via.placeholder.com/350x250/f28c33/fff',
+      ],
+      pages: [
+        {
+          name: 'Home page',
+          component: `<h1>Design Your Webview</h1>`,
+        },
+      ],
+    },
+  };
+
 export function Builder(){
-    return <h1>Builder</h1>
-}
\ No newline at end of file
+  
+    const onEditor = (editor) => {
+        // editor.setCSS('.sp-container button', {
+        //   'background-color': '#0000ff',
+        //   'color': '#fff',
+        //   'padding': '10px 20px',
+        //   'border': 'none',
+        //   'border-radius': '5px',
+        // });
+
+        editor.BlockManager.add('red-button-block', {
+          label: 'Red Button',
+          category: 'Styled Components',
+          content: `<button class="custom-red-button" style="background-color: red; color: white; padding: 10px 20px; border: none; border-radius: 5px; cursor: pointer;" contentEditable="true">Click Me</button>`,
+        });
+
+        editor.BlockManager.add('input-label-block', {
```

Comment:
I think that now we can customize the label block provided by Grapes, so let's please lose this block. 🙈